src/shogun/features/DenseFeatures.h

```diff
@@ -23,8 +23,6 @@
 #include <shogun/features/StringFeatures.h>
 #include <shogun/lib/DataType.h>
 
-#include <shogun/lib/SGMatrix.h>
```

Comment:
I believe I had placed that in there whilst trying to use EigenMatrixMapXt - I just took it out as I'm no longer using that.